### PR TITLE
Python 3 compatible

### PR DESCRIPTION
**Pull request by vangheem**

### CODE REVIEW

File: plone/app/drafts/proxy.py [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 from zope.component import adapter
-from zope.interface import implementedBy
-from zope.interface import implementer
-from zope.interface import providedBy
-from zope.interface.declarations import getObjectSpecification
-from zope.interface.declarations import ObjectSpecificationDescriptor
-
+from zope.interface import implementedBy, implementer, providedBy
+from zope.interface.declarations import (ObjectSpecificationDescriptor,
+                                         getObjectSpecification)
```
Comment (davisagli) [inline plone/app/drafts/proxy.py]: Why are you combining the imports? Plone style is to have a separate import for each name to avoid noisy diffs.

Comment (vangheem) [reply plone/app/drafts/proxy.py]: my bad, I ran isort on it but it didn't have isort configuration in setup.cfg--added.

**Comment by davisagli** [comment]
Looks okay now but I'm not sure what's up with the travis build.

**Comment by petschki** [comment]
I've fixed the hanging `createcoverage` job and enhanced the test matrix ... though I could need some help with the failing Plone 5.2 tests ...

**Comment by petschki** [comment]
@jensens can you release py3 compatibility fixes as 1.2.0 please?

**Pull request merged**